controllers/comments.controllers.js

```diff
@@ -0,0 +1,19 @@
+const {
+  removeCommentById,
+  checkCommentExists,
+} = require('../models/comments.models');
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  checkCommentExists(comment_id)
```

Comment:
you could achieve this with one function and a single SQL query, by `RETURNING *;` at the end of your DELETE query string, and then checking the rows for the deleted comment.